build.sh

```diff
@@ -225,6 +225,10 @@ while [[ $# -gt 0 ]]; do
         NO_JIT="-DNO_JIT_SH=1"
         ;;
 
+    --intl-icu)
```

Comment:
on the top of this file we list all the available args. This should also documented there. ++ it could be nice to have this param as `--with-intl` .. so, if user defines `--icu=...` and then `--with-intl`.. it might look nicer?